code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -65,7 +70,36 @@
 	new /obj/item/weapon/wirecutters(src)
 	new /obj/item/device/t_scanner(src)
 
+/obj/item/weapon/storage/belt/utility/chief
+	name = "advanced tool-belt"
+	desc = "The ancestral belt of Many-APCs-Charging, the original chief engineer from Space Native America. It's made out of the skins of the ancient enemy of engineers, giant spiders."
+	icon_state = "utilitychief"
+	item_state = "utilitychief"
+	w_class = 4
+	max_w_class = 3
+	storage_slots = 14
+	can_hold = list(
+		"/obj/item/weapon/crowbar",
+		"/obj/item/weapon/screwdriver",
+		"/obj/item/weapon/weldingtool",
+		"/obj/item/weapon/solder",
+		"/obj/item/weapon/wirecutters",
+		"/obj/item/weapon/wrench",
+		"/obj/item/device/multitool",
+		"/obj/item/device/flashlight",
+		"/obj/item/stack/cable_coil",
+		"/obj/item/device/t_scanner",
+		"/obj/item/device/analyzer",
+		"/obj/item/taperoll/engineering",
+		"/obj/item/weapon/extinguisher",
+		"/obj/item/weapon/rcd",
+		"/obj/item/weapon/pipe_dispenser",
+		"/obj/item/weapon/tile_painter",
+		"/obj/item/weapon/storage/component_exchanger"
+		)
 
+/obj/item/weapon/storage/belt/utility/chief/New()

```

Comment:
Most of the variables for the chief utility belt are also identical as their parent so they dont need to be redefined such as w_class, etc